From 6f057c203024eb9e13874e0643ca4beb03f093af Mon Sep 17 00:00:00 2001
From: Henrik Edin <henrik.edin@mongodb.com>
Date: Tue, 25 Apr 2023 12:17:29 +0000
Subject: [PATCH 14/28] SERVER-76473 Fix compiling S2 lib for Windows with
 VS2022

---
 src/third_party/s2/s2cellid.cc | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/src/third_party/s2/s2cellid.cc b/src/third_party/s2/s2cellid.cc
index 2cc7969172f..1140ca3ec8f 100644
--- a/src/third_party/s2/s2cellid.cc
+++ b/src/third_party/s2/s2cellid.cc
@@ -560,9 +560,3 @@ string S2CellId::slowToString() const {
 ostream& operator<<(ostream& os, S2CellId const& id) {
   return os << id.ToString();
 }
-
-#ifdef OS_WINDOWS
-template<> size_t stdext::hash_value<S2CellId>(const S2CellId &id) {
-    return static_cast<size_t>(id.id() >> 32) + static_cast<size_t>(id.id());
-}
-#endif
